### PR TITLE
br/stream: refactor the status command && print "no task" when no task here

### PR DESCRIPTION
**Pull request by YuJuncen**
Signed-off-by: Yu Juncen <yujuncen@pingcap.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/33410

Problem Summary:
There are some bugs after #33411:
- Nothing printed when there are no tasks.
- The global checkpoint ts doesn't be advanced.

### What is changed and how it works?
Those bugs are fixed, and some refactors are applied to the `status` command and the `ConsoleGlue` type.

### Check List

Tests 

- [x] Unit test (TBD)
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

```release-note
None
```

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]
LGTM

File: br/pkg/stream/stream_status.go [hunk 1]
```diff
@@ -232,3 +264,93 @@ func MaybeQPS(ctx context.Context, mgr *conn.Mgr) (float64, error) {
...
+		return nil, err
+	}
+	status, err := ctl.fillTask(ctx, *task)
+	if err != nil {
+		return nil, err
+	}
+	return []TaskStatus{status}, nil
+}
+
+func (ctl *StatusController) PrintToView(tasks []TaskStatus) {
```
Comment (joccau) [inline br/pkg/stream/stream_status.go:341]:
```suggestion
func (ctl *StatusController) printToView(tasks []TaskStatus) {
```

File: br/pkg/stream/stream_status.go [hunk 2]
```diff
@@ -232,3 +264,93 @@ func MaybeQPS(ctx context.Context, mgr *conn.Mgr) (float64, error) {
...
+	ctl.view.PrintTasks()
+}
+
+// PrintStatusOfTask prints the status of tasks with the name. When the name is *, print all tasks.
+func (ctl *StatusController) PrintStatusOfTask(ctx context.Context, name string) error {
+	tasks, err := ctl.getTask(ctx, name)
+	if err != nil {
+		return err
+	}
+	ctl.PrintToView(tasks)
```
Comment (joccau) [inline br/pkg/stream/stream_status.go:354]:
```suggestion
	ctl.printToView(tasks)
```

**Review by joccau — COMMENTED** [review]
rest LGTM

**Review by joccau — APPROVED** [review]

**Pull request merged**